bin/migrate-oats-data/applications/app_prep.py

```diff
@@ -233,6 +242,14 @@ def mapOatsToAlcsAppPrep(data):
 
     return data
 
+def mapOatsToAlcsLeg(data):
+
+    if data["legislation_code"]:
+        data["legislation_code"] = str(
+            OatsLegislationCodes[data["legislation_code"]].value
+        )
+
+    return data
 
 def get_update_query_for_nfu():
     query = """
```

Comment:
have you tried to reduce "query" duplication using the constants and custom functions as we discussed or is it in next MR?

Reply:
Was plan for next MR